### PR TITLE
Add React Renderers section

### PR DESCRIPTION
**Pull request by ilyalesik**

### CODE REVIEW

**Review by Haroenv — APPROVED** [review]
seems good, why did you remove ink from the list? it seems like a renderer to me :)

File: README.md [hunk 1]
```diff
@@ -396,6 +397,15 @@ A collection of awesome things regarding the React ecosystem.
 - [React 360](https://facebook.github.io/react-360/) - Create exciting 360 and VR experiences using React
```
Comment (Haroenv) [inline README.md:397]: this and react native should be added to the list as well

**Comment by ilyalesik** [comment]
@Haroenv thanks!

> why did you remove ink from the list? it seems like a renderer to me :)

I just got a Travis error that Ink already in the list:

Screenshot 2020-11-28 at 18 00 53

Should I replace Ink to the Renderers section? Another scenario is adding react renderers to specific sections - e.g. Design Editors - react-figma, react-sketchapp, 3D - react-tree-fiber, react-360, ...

**Comment by Haroenv** [comment]
I think it's best if they're repeated, you can add the specific links that you put twice in -w here: https://github.com/enaqx/awesome-react/blob/master/.travis.yml#L7

**Comment by ilyalesik** [comment]
@Haroenv done!

**Comment by Haroenv** [comment]
thanks! you can add react native in a subsequent PR

**Pull request merged**